wallet/res/layout/send_coins_fragment1.xml

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file used?  It has a 1 in the name.

Reply:
File removed